### PR TITLE
Try to further improve the english orthography rules

### PR DESCRIPTION
**Pull request by samlh**
Inspired by issue #138

Warning: I wrote this on a machine without python, so it may be entirely broken

### CODE REVIEW

File: plover/test_orthography.py [hunk 1]
```diff
@@ -97,7 +97,14 @@ def test_add_suffix(self):
             ('white', 'er', 'whiter'),
             ('crap', 'y', 'crappy'),
             ('lad', 'er', 'ladder'),
-        
+            ('translucent', 'cy', 'translucency'),
+            ('bankrupt', 'cy', 'bankruptcy'),
+            ('inadequate', 'cy', 'inadequatecy'), # TODO: inadequacy
+            ('secret', 'cy', 'secretcy'), # TODO: secrecy
```
Comment (morinted) [inline plover/test_orthography.py]: Would you like to finish your TODOs before this pull request is tested?

**Comment by morinted** [comment]
Can you confirm what the purpose of your TODOs are? Would you like to handle those cases?

**Comment by samlh** [comment]
Sorry, I had totally forgotten this. I've pushed a 2 commits to remove the remaining TODOs - I think this should be good to go.

**Comment by morinted** [comment]
Thanks @samlh, it all checks out.

**Pull request merged**